### ISSUE TITLE
[a11y][ui5-button]The hover state of ui5-button must meet 7:1 against the black background in high contrast black mode

### ISSUE THREAD

**Issue by Hubery-Shen** [SAP/ui5-webcomponents#2432]
**Dear colleagues,**

I am from SuccessFactors Web Experience Shanghai team.

**Bug Description**

The hover state of ui5-button must meet 7:1 against the black background in high contrast black mode
![button text and background color](https://user-images.githubusercontent.com/63704163/98085068-c3566680-1eb7-11eb-8c7e-12507fa69e82.png)

The current ratio is: 6.99:1, and it should be 7:1

**Comment by fifoosid** [comment]
Hi @Hubery-Shen 

Is this issue empty by mistake?

**Comment by Hubery-Shen** [comment]
> Hi @Hubery-Shen
> 
> Is this issue empty by mistake?

I am editing this issue.

**Comment by ilhan007** [comment]
Hello @SAP/ui5-webcomponents-topic-b we have this contrast ratio issue for the ui5-button on hover in HCB theme
between the white border(#ffffff) and the hover background (#7a5100).

The same colors, that are defined by the designers, are used in the sap.m.Button in HCB
https://openui5nightly.hana.ondemand.com/entity/sap.m.Button/sample/sap.m.sample.Button

However, the author's tool shows 6.99 : 1 contrast ratio (just below 7:1), but It's possible that we use different tools. Could you confirm and prove that the border and hover colors are fine?

**Comment by Hubery-Shen** [comment]
Hi @ilhan007 

![Screen Shot 2020-11-05 at 5 49 09 PM](https://user-images.githubusercontent.com/63704163/98225099-43033480-1f8f-11eb-8c19-7efb61373f4b.png)

We use **Colour Contrast Analyser** tool to test this.
I can confirm and prove that the border and hover colors are fine

**Comment by ilhan007** [comment]
@Hubery-Shen are you comparing the hover background and the border color? 
Should not you compare the border color (white) and the black background? Then the ration will meet the 7:1 ratio for sure.

**Comment by ilhan007** [comment]
closing due to inactivity
